abipy/lessons/lesson_helper_functions.py

```diff
@@ -8,10 +8,34 @@
 import sys
 import os
 import shutil
+import yaml
+import html2text
+from variables import *
 import abipy.abilab as abilab
 import abipy.data as abidata
 
 
+class VariableDatabase(object):
```

Comment:
Use a singleton or 

_VAR_DATABASE = None

def get_database():
  global _VAR_DATABASE
  if _VAR_DATABASE is None: _VAR_DATABASE = load_from_file(absolute_path)
  return _VAR_DATABASE